packages/ui/components/cards/FeedCard.vue

```diff
@@ -104,6 +106,17 @@ export default {
 a {
   color: var(--value-color);
 }
+.card-border {
+  &.operational {
+    border: var(--card-border);
+  }
+  &.delay {
+    border: var(--delay-status-border);
+  }
+  &.error {
+    border: var(--error-status-border);
+  }
+}
```

Comment:
Is this being used?

Reply:
Yes, when there is a delay the border is yellow